src/main/resources/application-local.yml

```diff
@@ -5,3 +5,6 @@ network-store-server:
   base-uri: http://localhost:8080
 report-server:
   base-uri: http://localhost:5028
+
+loadflow:
+  default-provider: OpenLoadFlow
```

Comment:
Missing newline

Reply:
property moved to application.yml with new line